examples/jenkins/pipeline/nodejs-sample-pipeline.yaml

```diff
@@ -71,11 +71,8 @@ spec:
                             openshift.withCluster() {
                                 openshift.withProject() {
                                     def builds = openshift.selector("bc", templateName).related('builds')
-                                    // wait up to 5 minutes for the build to complete
-                                    timeout(5) {
-                                        builds.untilEach(1) {
-                                            return (it.object().status.phase == "Complete")
-                                        }
+                                    builds.untilEach(1) {
+                                        return (it.object().status.phase == "Complete")
```

Comment:
is there an overall timeout in place?  this and the DC change below make me worry when things go bad, we're going to hit hte 6 hour timeout before it fails.